app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -129,18 +147,34 @@
   }
 }
 
-.searchResultContent {
+.searchResultSubcatContainer {
   display: flex;
-  gap: 25px;
-  justify-items: stretch;
+  align-items: flex-start;
+  gap: var(--general-spacing-xxs, 4px);
 
-  @media screen and (max-width: $break-desktop) {
-    gap: 32px;
-    grid-template-columns: none;
-    justify-items: start;
+  @media screen and (max-width: $break-tablet-s) {
+    margin-top: $general-spacing-s;
   }
 }
 
+.searchResultSubcat,
+.searchResultSubcatTooltip {
+  border-radius: var(--rounded-full, 9999px);
```

Comment:
Oh, interesting hack. Just read about it.

Reply:
oops actually, I should change this to a var. I copied and pasted from figma.